CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [1.0.3](https://github.com/fiatconnect/fiatconnect-types/compare/v1.0.0...v1.0.3) (2022-03-11)
+
+Fix an issue with PersonalDataAndDocumentsKyc being exported under the wrong name.
+
 ### [1.0.2](https://github.com/fiatconnect/fiatconnect-types/compare/v1.0.0...v1.0.2) (2022-03-11)
```

Comment:
I'm just noticing this now, but it looks like the semver was done incorrectly here. When a new type is added, that's a minor version, not a patch. When a type is removed, that's a major version since it's not backwards compatible. https://semver.org/

Reply:
we may be able to delete the versions that were published if it's been less than 24hrs